content/episodes/113-multimedia-no-linux-part-2.rst

```diff
@@ -0,0 +1,176 @@
+Episódio 113: Multimedia no Linux - Parte 2
+###########################################
+:date: 2017-08-14
+:authors: Og Maciel, Elyézer Rezende, Bruno Rocha
+:category: Podcast
+:podcast: https://archive.org/download/castalio-podcast-113/castalio-podcast-113.mp3
+:tags: danilo j. s. bellini, audiolazy, ardour, audacity, AviX, Reaper, Adobe Audition, 
+       LMMS, Hydrogen, Fruit Loops, Pro Tools, Sox, ffmpeg, Csound, Pure Data, Faust, 
+       SuperCollider, War Eternal, Arch Enemy, Sacred Power of Raging Winds, Rhapsody, 
+       Aces High, Iron Maiden, Erotomania, Dream Theater, The Call of the Mountains, 
+       Eluveitie, Wikipedia, Game of Thrones, Vikings, One Piece, Shōnen, Seinen, Dexter
+:description: Produção e edição de áudio no Linux com Danilo J. S. Bellini
+:image: images/epiode-113.png
+:image-alt: Multimedia no Linux - Parte 2
+
+Olá pessoal e sejam bem-vindos à mais um episódio do Castálio Podcast!
+
+No Episódio de hoje daremos continuidade a nossa série sobre produção multimídia no
+Linux. Já falamos um pouco sobre `GIMP e Inkscape`_, e teremos ainda mais episódios
+sobre produção gráfica, 3D com Blender e edição de vídeo. Mas hoje a nossa atenção
+estará voltada para um tema pouco falado e que é muito importante também para produção
+de vídeo, produção musical e é claro para edição de podcasts.
+
+Para falar sobre produção e edição de áudio no Linux hoje trouxemos o Danilo J. S. 
+Bellini que é engenheiro eletricista, bacharel em música e mestre em ciência da computação
+pela USP. Ele é também o criador do `AudioLazy`_, uma biblioteca de áudio escrita em Python
+que te permite fazer um monte de coisas bacanas na hora de editar áudio.
```

Comment:
Nao sei se editar audio seria o caso uma vez que a AudioLazy e "Real-Time Expressive Digital Signal Processing (DSP) Package for Python!"